app/components/records-charter.js

```diff
@@ -3,29 +3,33 @@
  *  Licensed under the terms of the Apache License, Version 2.0.
  *  See the LICENSE file associated with the project for terms.
  */
-import { A } from '@ember/array';
+import { WINDOW_CREATED_KEY, WINDOW_NUMBER_KEY } from 'bullet-ui/components/result-viewer';
 
+import { A } from '@ember/array';
 import { isEmpty, isEqual, typeOf } from '@ember/utils';
 import { computed } from '@ember/object';
 import { not, alias, or } from '@ember/object/computed';
 import Component from '@ember/component';
+import { inject as service } from '@ember/service';
 
 export default Component.extend({
+  moment: Ember.inject.service(),
```

Comment:
moment: service(),

actually moment is never used in your PR.

Reply:
Will remove